src/SldStyleParser.ts

```diff
@@ -388,43 +371,83 @@ export class SldStyleParser implements StyleParser<string> {
    * @return {string} The name to be used for the GeoStyler Style Style
    */
   getStyleNameFromSldObject(sldObject: any): string {
-    const userStyleTitle = _get(sldObject, 'StyledLayerDescriptor.NamedLayer[0].UserStyle[0].Title[0]._');
-    const namedLayerName = _get(sldObject, 'StyledLayerDescriptor.NamedLayer[0].Name[0]._');
+    const userStyleTitle = get(sldObject, 'StyledLayerDescriptor.NamedLayer[0].UserStyle.Title.#text');
+    const namedLayerName = get(sldObject, 'StyledLayerDescriptor.NamedLayer.Name.#text');
     return userStyleTitle ? userStyleTitle
       : namedLayerName ? namedLayerName : '';
   }
 
   /**
-   * Creates a GeoStyler-Style StrMatchesFunctionFilterr from a SLD strMatches Function.
+   * Get the GeoStyler-Style Filter from an SLD Rule.
```

Comment:
```suggestion
   * Get the GeoStyler-Style Filter from a SLD Rule.
```

`Filter` also maybe lowercase?